### DISCUSSION TITLE
Georeferenced videos

### DISCUSSION THREAD

**Discussion by chrleon** [maplibre/maplibre#368] [Q&A]
I'm not sure if this is the correct place for it, but in the [Maplibre docs](https://maplibre.org/maplibre-gl-js/docs/examples/video-on-a-map/) there is a page on adding a video to the map. 

Is that video geoplaced in a way, or is it trial and error until it fit? The docs are not saying much about this. Thanks.

**Comment by HarelM** [comment]
There are coordinates to the video source...

**Reply by chrleon** [reply]
Yes I see those, thanks.  However, I was wondering how these coordinates would be decided. Like how could I project a movie like that to be so accurate on my own maps. 

Sorry for not managing to be clear enough.

**Reply by HarelM** [reply]
Much like any other feature I guess, you need to know where to place it... But I can understand there's no good source of that data...
